policies/ecc-azure-420-asb_deprecated_java_funcapp.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2023 EPAM Systems, Inc.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+policies:
+  - name: ecc-azure-420-asb_deprecated_java_funcapp
+    comment: '020056170000'
+    description: |
+      Function app has an deprecated Java version
+    resource: azure.webapp
+    filters:
+      - type: value
+        key: kind
+        value: functionapp*
+        op: regex
+      - or:
+        - type: configuration
+          key: linuxFxVersion
+          value_regex: ".*JAVA.([0-9]{1})$"
+          op: lt
+          value: "8"
+        - and:
+          - type: configuration
+            key: javaVersion
+            value: "1.8"
+            op: lt
+          - type: configuration
+            key: javaVersion
+            value: ""
```

Comment:
do we need this check? can empty string match condition "lt 1.8"?

Reply:
@anna-shcherbak, If we don't specify this filter, it would take any other function app where this field is "". Because this field is used only WINDOWS based function app and If the customer will have linux function app this row would be "".